docs/en-us/getting-started/access-and-setup-vsce.md

```diff
@@ -7,6 +7,61 @@
 
 One of the clients that give you access to COSMO Alpaca is the VS Code Extension. Before setting it up you might want to [create an organization](create-org.md) beforehand.
 
+# [**vscode.dev (Browser Version)**](#tab/vscode.dev)
+
+## COSMO Alpaca – Browser Edition
+
+With the **browser-based version** of Visual Studio Code available at [vscode.dev](https://vscode.dev), you can use **COSMO Alpaca** without installing VS Code locally. This is particularly useful for:
+
+1. **Developers** – a lightweight, fast environment.
```

Comment:
I think developers should be last here, add a recommendation to use the local VS Code for devs